### DISCUSSION TITLE
[Feature Request] Support multi-line snippet

### DISCUSSION THREAD

**Discussion by jiamingz42** [knqyf263/pet#284] [Ideas]

**Comment by nfedyashev** [comment]
That would be nice indeed!

What I noticed is that if you manually add "\n" to your command in ".config/pet/snippet.toml" then those snippets outputs as a multi-line command instead.

Still investigating why I can't use "native" multi-line input for new pet snippets. It would have been really cool if `pet` respected `editor` setting in `.config/pet/config.toml` and used a specified editor instead of a single-line input that it currently has.

**Comment by ofey404** [comment]
I've made a PR for multi-line support #161. Basic feature is usable.
